src/analyze/analyze-verify.h

```diff
@@ -14,6 +14,8 @@ typedef enum RecursiveErrors {
         _RECURSIVE_ERRORS_INVALID = -EINVAL,
 } RecursiveErrors;
 
+int generate_path(char **var, char **filenames);
+int prepare_filename(const char *filename, char **ret);
```

Comment:
if you export these, please maybe prefix with `verify_` or so, to make clear what they belong to